data/campaigns/World_Conquest/lua/game_mechanics/wocopedia/help.lua

```diff
@@ -53,19 +53,19 @@ function wesnoth.wml_actions.wc2_show_wocopedia(cfg)
+			_ "For feedback please either post in the World Conquest II thread in the official Wesnoth forum https://r.wesnoth.org/t39651 or file an issue at GitHub https://github.com/gfgtdf/World_Conquest_II/issues .\n\n" ..
 			""
 		local str_cat_abilities = _ "Abilities"
 		local str_des_abilities =
 			_ "Ability <b>Autorecall</b>:\n" ..
-			_ "Units with trait HEROIC are recalled at start of each scenario with no cost (up to castle size).\n\n" ..
+			_ "Units with trait HEROIC are recalled at the start of each scenario with no cost (up to castle size).\n\n" ..
 			""
 		local str_cat_training = _ "Training"
-		local str_des_training = _ "<b>Training</b>\nTraining improves newly recruited units, it has no effect on already recruited units. The follwing list shows all available trainings, the training you currently have is marked in green."
+		local str_des_training = _ "<b>Training</b>\nTraining improves newly recruited units, it has no effect on already recruited units. The following list shows all available training, the training you currently have is marked in green."
 		local str_cat_items = _ "Artifacts"
-		local str_des_items = _ "<b>Items</b>\nItems can be given to units to make them stronger. You can get artifcats in three ways: 1) By choosing an item as your starting bonus, 2) By finding it on a map in a bonus point, 3) By dropping from enemies in later scenarios. Note however that not all units can pickup all items."
+		local str_des_items = _ "<b>Items</b>\nItems can be given to units to make them stronger. You can get artifcats in three ways: 1) By choosing an item as your starting bonus, 2) By finding it on a map in a bonus point, 3) By dropping from enemies in later scenarios. Note, however, that not all units can pick up all items."
 		local str_cat_era = _ "Factions"
-		local str_des_era = _ "<b>Factions</b>\n The Word Conquest 2 era consists of faction that are build of pairs of mainline faction of which at one has a healer available (Drakes, Rebels and Loyalists), and one does not (Orcs, Dwarves and Undead) the recruilist is also organized in pairs so that you sometimes have to recruit a different units before you can recruit the units that you want. The available heroes, desertes and random leaders also depend on your factions, the items you can get do not depend on the faction you choose."
+		local str_des_era = _ "<b>Factions</b>\n The World Conquest II era consists of factions that are build of pairs of mainline faction of which at one has a healer available (Drakes, Rebels and Loyalists), and one does not (Orcs, Dwarves and Undead) the recruit list is also organized in pairs so that you sometimes have to recruit a different unit before you can recruit the units that you want. The available heroes, desertes, and random leaders also depend on your factions, the items you can get do not depend on the faction you choose."
```

Comment:
I think the 2/II should be removed in all user-facing strings.

Reply:
I recall the discussion in #wesnoth-dev about WC vs WC2 but my focus here is simply on updating things that seem/look nonsensical to me. Mainly spelling errors but also things that I just didn't understand as an English speaker. (As an aside, WC2 means something completely different to me altogether.)